### PR TITLE
First update of KEB docs

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Initial update of KEB docs
- Add deprecation notes
- Consistent use of the name "SAP BTP, Kyma runtime" or "Kyma runtime"
- Add the repository README file
- Formatting and language fixes
- Add `keb-cronjobs.md`
- Add the KEB target architecture

**Related issue(s)**
See #26

### CODE REVIEW

File: docs/01-01-overview.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Kyma Environment Broker (KEB) overview
 
-Kyma Environment Broker is a component that allows you to provision Kyma as a Runtime on clusters provided by third-party providers. It uses the Runtime Provisioner's API to install Kyma on a given cluster.
+Kyma Environment Broker is a component that allows you to provision Kyma as a [SAP BTP, Kyma runtime](https://kyma-project.io/#/?id=kyma-and-sap-btp-kyma-runtime) on clusters provided by third-party providers. It uses the Runtime Provisioner's API to install Kyma on a given cluster.
```
Comment (piotrmiskiewicz) [inline docs/01-01-overview.md:3]: KEB uses Provisioner to create a cluster, then uses Reconciler and LM (the target architecture - only LM) to install Kyma there

File: docs/03-02-runtime-components.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
    * `kyma-installer-cluster.yaml` file with the given Kyma version
    * `managed-offering-components.yaml` file with additional managed components that are added at the end of the base components list
 
-2. The user provisions a Runtime and selects optional components that they want to install.
+2. The user provisions a Kyma runtime and selects optional components that they want to install.
 
 3. KEB composes the final list of components by removing components that were not selected by the user. It also adds the proper global and components overrides and sends the whole provisioning information to the Runtime Provisioner.
 
-There is a defined [list of the component names](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/runtime/components). Use these names in your implementation.
+There is a defined [list of the components and their names](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/runtime/components). Use these names in your implementation.
```
Comment (grego952) [inline docs/03-02-runtime-components.md:18]:
```suggestion
There is a defined [list of the components and their names](https://github.com/kyma-project/kyma-environment-broker/blob/main/internal/runtime/components/components.go). Use these names in your implementation.
```

File: docs/03-02-runtime-components.md [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
 
 ## Disabled components
```
Comment (grego952) [inline docs/03-02-runtime-components.md:20]: The link to the disable components, in line 22, is also broken. It should probably lead to https://github.com/kyma-project/kyma-environment-broker/blob/main/internal/runtime/disabled_components.go

File: docs/03-02-runtime-components.md [hunk 4]
```diff
@@ -40,14 +42,14 @@ runtime.NewGenericComponentDisabler("component-name", "component-namespace")
 * If disabling a given component requires more complex logic, create a new file called `internal/runtime/{compoent-name}_disabler.go` and implement a service which fulfills the following interface:
```
Comment (grego952) [inline docs/03-02-runtime-components.md:42]:
```suggestion
* If disabling a given component requires more complex logic, create a new file called `internal/runtime/{component-name}_disabler.go` and implement a service that fulfills the following interface:
```

File: docs/03-02-runtime-components.md [hunk 5]
```diff
@@ -40,14 +42,14 @@ runtime.NewGenericComponentDisabler("component-name", "component-namespace")
...
 
 ```go
-// OptionalComponentDisabler disables component form the given list and returns a modified list
+// OptionalComponentDisabler disables component from the given list and returns a modified list
 type OptionalComponentDisabler interface {
 	Disable(components internal.ComponentConfigurationInputList) internal.ComponentConfigurationInputList
 ```
 
->**NOTE**: Check the [CustomDisablerExample](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/runtime/custom_disabler_example.go) as an example of custom service for disabling components.
+>**TIP**: Check the [CustomDisablerExample](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/runtime/custom_disabler_example.go) as an example of custom service for disabling components.
```
Comment (grego952) [inline docs/03-02-runtime-components.md:50]:
```suggestion
>**TIP**: Check the [CustomDisablerExample](https://github.com/kyma-project/kyma-environment-broker/blob/main/internal/runtime/custom_disabler_example.go) as an example of custom service for disabling components.
```

File: docs/03-02-runtime-components.md [hunk 6]
```diff
@@ -62,6 +64,6 @@ optionalComponentsDisablers := runtime.ComponentsDisablers{
 }
 ```
 
-### Remove the optional component
+### Remove an optional component from the disabled components list
 
-If you want to remove the option to disable components and make them required during Kyma installation, remove a given entry from the **optionalComponentsDisablers** list in the [`cmd/broker/main.go`](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/cmd/broker/main.go) file.
+If you want to remove the option to disable components and make them required during SAP BTP, Kyma runtime installation, remove a given entry from the **optionalComponentsDisablers** list in the [`cmd/broker/main.go`](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/cmd/broker/main.go) file.
```
Comment (grego952) [inline docs/03-02-runtime-components.md:69]:
```suggestion
If you want to remove the option to disable components and make them required during SAP BTP, Kyma runtime installation, remove a given entry from the **optionalComponentsDisablers** list in the [`cmd/broker/main.go`](https://github.com/kyma-project/kyma-environment-broker/blob/main/cmd/broker/main.go) file.
```

File: docs/03-03-runtime-operations.md [hunk 7]
```diff
@@ -9,23 +9,23 @@ Kyma Environment Broker allows you to configure operations that you can run on a
...
+Each provisioning step is responsible for a separate part of preparing Kyma runtime parameters. For example, in a step you can provide tokens, credentials, or URLs to integrate SAP BTP, Kyma runtime with external systems. All data collected in provisioning steps are used in the step called [`create_cluster_configuration`](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/process/provisioning/create_cluster_configuration.go) which transforms the data into a request input. The request is sent to the Runtime Provisioner component which provisions a Kyma runtime.
```
Comment (grego952) [inline docs/03-03-runtime-operations.md:12]:
```suggestion
Each provisioning step is responsible for a separate part of preparing Kyma runtime parameters. For example, in a step, you can provide tokens, credentials, or URLs to integrate SAP BTP, Kyma runtime with external systems. All data collected in the provisioning steps are used in the step called [`create_cluster_configuration`](https://github.com/kyma-project/kyma-environment-broker/blob/main/internal/process/provisioning/create_cluster_configuration.go) which transforms the data into a request input. The request is sent to the Runtime Provisioner component which provisions a Kyma runtime.
```

File: docs/03-03-runtime-operations.md [hunk 8]
```diff
@@ -9,23 +9,23 @@ Kyma Environment Broker allows you to configure operations that you can run on a
...
 | check_runtime  | Check_Runtime                      | Provisioning             | Checks the status of the Provisioner process and asks the Director for the Dashboard URL if the provisioning is completed in Gardener.      | Team Gopher     |
 | create_runtime | Get_Kubeconfig                     | Provisioning             | Gets the kubeconfig.                                                                                                                        | Team Gopher     |
-| create_runtime | Inject_BTP_Operator_Credentials    | Provisioning             | Creates a secret in the SKR with credentials for BTP.                                                                                         | Team Gopher     |
+| create_runtime | Inject_BTP_Operator_Credentials    | Provisioning             | Creates a secret in the Kyma runtime with credentials for BTP.                                                                                         | Team Gopher     |
```
Comment (grego952) [inline docs/03-03-runtime-operations.md:28]:
```suggestion
| create_runtime | Inject_BTP_Operator_Credentials    | Provisioning             | Creates a Secret in the Kyma runtime with credentials for BTP.                                                                                         | Team Gopher     |
```
Change to uppercase if it's a Kubernetes Secret.

File: docs/03-03-runtime-operations.md [hunk 9]
```diff
@@ -59,7 +59,7 @@ The deprovisioning process contains the following steps:
 
 ## Upgrade Kyma
 
-Each upgrade step is responsible for a separate part of upgrading Runtime dependencies. To properly upgrade the Runtime, you need the data used during the Runtime provisioning. You can fetch this data from the **ProvisioningOperation** struct in the [initialization](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/process/upgrade_kyma/initialisation.go) step.
+Each upgrade step is responsible for a separate part of upgrading Kyma runtime dependencies. To properly upgrade SAP BTP, Kyma runtime, you need the data used during the provisioning. You can fetch this data from the **ProvisioningOperation** struct in the [initialization](https://github.com/kyma-project/control-plane/blob/main/components/kyma-environment-broker/internal/process/upgrade_kyma/initialisation.go) step.
```
Comment (grego952) [inline docs/03-03-runtime-operations.md:62]:
```suggestion
Each upgrade step is responsible for a separate part of upgrading Kyma runtime dependencies. To properly upgrade SAP BTP, Kyma runtime, you need the data used during the provisioning. You can fetch this data from the **ProvisioningOperation** struct in the [initialization](https://github.com/kyma-project/kyma-environment-broker/blob/main/internal/process/upgrade_kyma/initialisation.go) step.
```

File: docs/03-05-authorization.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # Authorization
 
-Kyma Environment Broker provides OAuth2 authorization. For this purpose, Kyma Environment Broker uses the [ApiRule](https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-custom-resources/apix-01-apirule/) custom resource which generates a [VirtualService](https://istio.io/docs/reference/config/networking/virtual-service/) and uses  [Oathkeeper Access Rules](https://www.ory.sh/docs/oathkeeper/api-access-rules) to allow or deny access.
+Kyma Environment Broker (KEB) provides OAuth2 authorization. For this purpose, KEB uses the [ApiRule](https://kyma-project.io/docs/kyma/latest/05-technical-reference/00-custom-resources/apix-01-apirule/) custom resource which generates a [VirtualService](https://istio.io/docs/reference/config/networking/virtual-service/) and uses  [Oathkeeper Access Rules](https://www.ory.sh/docs/oathkeeper/api-access-rules) to allow or deny access.
```
Comment (grego952) [inline docs/03-05-authorization.md:3]:
```suggestion
Kyma Environment Broker (KEB) provides OAuth2 authorization. For this purpose, KEB uses the [ApiRule](https://kyma-project.io/#/05-technical-reference/00-custom-resources/apix-01-apirule) custom resource which generates a [VirtualService](https://istio.io/docs/reference/config/networking/virtual-service/) and uses  [Oathkeeper Access Rules](https://www.ory.sh/docs/oathkeeper/api-access-rules) to allow or deny access.
```

File: docs/03-05-authorization.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
...
 To authorize with the Kyma Environment Broker, use an OAuth2 client registered through the [Hydra Maester controller](https://github.com/ory/k8s/blob/master/docs/helm/hydra-maester.md).
```
Comment (grego952) [inline docs/03-05-authorization.md:4]:
```suggestion
To authorize with KEB, use an OAuth2 client registered through the [Hydra Maester controller](https://github.com/ory/k8s/blob/master/docs/helm/hydra-maester.md).
```

File: docs/03-05-authorization.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
...
 
 To access the Kyma Environment Broker endpoints, use the `/oauth` prefix before OSB API paths. For example:
```
Comment (grego952) [inline docs/03-05-authorization.md:6]:
```suggestion
To access the KEB endpoints, use the `/oauth` prefix before OSB API paths. For example:
```

File: docs/03-02-runtime-components.md [hunk 13]
```diff
@@ -1,19 +1,21 @@
-# Runtime components
+# SAP BTP, Kyma runtime components
 
-Kyma Environment Broker (KEB) serves the functionality of composing the list of components that are installed in a Runtime. The diagram and steps describe the KEB workflow in terms of calculating and processing Runtime components:
+> **NOTE:** Once all Kyma components become independent modules, KEB will no longer be required to send components to Reconciler and this document will be deprecated as irrelevant.
```
Comment (grego952) [inline docs/03-02-runtime-components.md:3]:
```suggestion
> **NOTE:** Once all Kyma components become independent modules, Kyma Environment Broker (KEB) will no longer be required to send components to Reconciler and this document will be deprecated as irrelevant.
```

File: docs/03-06-runtime-overrides.md [hunk 14]
```diff
@@ -1,6 +1,8 @@
-# Set overrides for Kyma Runtime
+# Set overrides for SAP BTP, Kyma runtime
 
-You can set overrides to customize your Kyma Runtime. To provision a cluster with custom overrides, add a Secret or a ConfigMap with a specific label. Kyma Environment Broker uses this Secret and/or ConfigMap to prepare a request to the Runtime Provisioner.
+> **NOTE:** Once all Kyma components become independent modules, KEB will no longer be required to send overrides to Reconciler and this document will be deprecated as irrelevant.
```
Comment (grego952) [inline docs/03-06-runtime-overrides.md:3]:
```suggestion
> **NOTE:** Once all Kyma components become independent modules, Kyma Environment Broker will no longer be required to send overrides to Reconciler and this document will be deprecated as irrelevant.
```

File: docs/03-07-environments-cleanup.md [hunk 15]
```diff
@@ -1,13 +1,15 @@
...
+
+Environments Cleanup CronJob removes Kyma Environments which are older than 24h. The CronJob is scheduled to run daily at midnight local time defined in the system.
 
 ## Prerequisites
 
 Environments Cleanup requires access to:
 - Gardener project of choice to filter Shoots without a proper label
-- Database to get an Instance ID for each Runtime marked for deletion
-- Kyma Environment Broker to trigger Runtime deprovisioning
+- the KEB database to get an Instance ID for each SAP BTP, Kyma runtime marked for deletion
```
Comment (grego952) [inline docs/03-07-environments-cleanup.md:11]:
```suggestion
- the Kyma Environment Broker (KEB) database to get an Instance ID for each SAP BTP, Kyma runtime marked for deletion
```
Is this KEB something else than the following one? If yes, please discard this ant the following suggestion.

File: docs/03-07-environments-cleanup.md [hunk 16]
```diff
@@ -1,13 +1,15 @@
...
+- Kyma Environment Broker (KEB) to trigger Kyma runtime deprovisioning
```
Comment (grego952) [inline docs/03-07-environments-cleanup.md:12]:
```suggestion
- KEB to trigger Kyma runtime deprovisioning
```

File: docs/03-08-kyma-versions.md [hunk 17]
```diff
@@ -1,6 +1,8 @@
 # Configure Kyma version
 
-Kyma Environment Broker is configured with a default Kyma version specified in the **APP_KYMA_VERSION** environment variable. This means that each Kyma Runtime provisioned by Kyma Environment Broker in a given global account is installed in the default Kyma version.
+> **NOTE:** Once all Kyma components become independent modules, KEB will no longer be required to send components to Reconciler and this document will be deprecated as irrelevant.
```
Comment (grego952) [inline docs/03-08-kyma-versions.md:3]:
```suggestion
> **NOTE:** Once all Kyma components become independent modules, Kyma Environment Broker (KEB) will no longer be required to send components to Reconciler and this document will be deprecated as irrelevant.
```

File: docs/03-08-kyma-versions.md [hunk 18]
```diff
@@ -1,6 +1,8 @@
...
+
+Kyma Environment Broker is configured with a default Kyma version specified in the **APP_KYMA_VERSION** environment variable. This means that each SAP BTP, Kyma runtime provisioned by Kyma Environment Broker in a given global account is installed in the default Kyma version.
```
Comment (grego952) [inline docs/03-08-kyma-versions.md:5]:
```suggestion
Kyma Environment Broker (KEB) is configured with a default Kyma version specified in the **APP_KYMA_VERSION** environment variable. This means that each SAP BTP, Kyma runtime provisioned by KEB in a given global account is installed in the default Kyma version.
```

File: docs/03-10-orchestration.md [hunk 19]
```diff
@@ -1,6 +1,8 @@
...
-Orchestration is a mechanism that allows you to upgrade Kyma Runtimes. To create an orchestration, [follow this tutorial](08-05-orchestrate-kyma-upgrade.md). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma Runtimes) in the Gardener cluster and narrows them to the IDs that you have specified in the request body. Then, `KymaUpgradeManager` performs the [upgrade steps](03-03-runtime-operations.md#upgrade) logic on the selected Runtimes.
+> **NOTE:** Once all Kyma components become modules, they will be upgraded independently of one another and of Kyma Environment Broker (KEB). This document will be deprecated as irrelevant.
+
+Orchestration is a mechanism that allows you to upgrade SAP BTP, Kyma runtimes. To create an orchestration, [follow this tutorial](08-05-orchestrate-kyma-upgrade.md). After sending the request, the orchestration is processed by `KymaUpgradeManager`. It lists Shoots (Kyma runtimes) in the Gardener cluster and narrows them to the IDs that you have specified in the request body. Then, `KymaUpgradeManager` performs the [upgrade steps](03-03-runtime-operations.md#upgrade) logic on the selected Kyma runtimes.
```
Comment (grego952) [inline docs/03-10-orchestration.md:5]: There is no `upgrade` anchor in the second link. Did you mean `update`?

File: docs/03-15-trial-cleanup-cronjob.md [hunk 20]
```diff
@@ -11,13 +11,13 @@ If the instance is already in the `suspended` state, this instance is just marke
 ### Dry-run mode
 
 If you need to test the Job, you can run it in the `dry-run` mode.
-In that mode, the Job only logs the information about the candidate instances (i.e. instances meeting the configured criteria). The instances are not affected.
+In that mode, the Job only logs the information about the candidate instances (that is, instances meeting the configured criteria). The instances are not affected.
```
Comment (grego952) [inline docs/03-15-trial-cleanup-cronjob.md:14]:
```suggestion
In that mode, the Job only logs the information about the candidate instances that is, instances meeting the configured criteria. The instances are not affected.
```

File: docs/03-16-deprovision-retrigger-cronjob.md [hunk 21]
```diff
@@ -23,9 +23,8 @@ By default, the CronJob is set to run every day at 3:00 am:
 kyma-environment-broker.trialCleanup.schedule: "0 3 * * *"
 ```
 
-> **NOTE**
-> If you need to test the Job, you can run it in the dry-run mode.
-> In this mode, the Job only logs the information about the candidate instances (i.e. instances meeting the configured criteria). The instances are not affected.
+> **NOTE:** If you need to test the Job, you can run it in the `dry-run` mode.
+> In this mode, the Job only logs the information about the candidate instances (that is, instances meeting the configured criteria). The instances are not affected.
```
Comment (grego952) [inline docs/03-16-deprovision-retrigger-cronjob.md:27]:
```suggestion
> In this mode, the Job only logs the information about the candidate instances that is, instances meeting the configured criteria. The instances are not affected.
```

File: docs/03-17-runtime-reconciler.md [hunk 22]
```diff
@@ -1,12 +1,18 @@
...
+Runtime Reconciler is an application that performs reconciliation tasks on SAP BTP, Kyma runtime.
 
-## Overview
+## Details
 
-Currently, there is one task for Runtime Reconciler. It reconciles BTP Manager Secrets on Kyma Runtimes. It can do it in two ways: 
-- by implementation with the usage of Runtime Watcher, which sends events about changes of the Secret from Kyma Runtime to KEB in real time. 
-- with a job, which periodically loops over all instances from the KEB database; each instance has an existing assigned Runtime ID; the job checks if the Secret on the Kyma Runtime matches the credentials from the KEB database.
+Currently, there is one task for Runtime Reconciler. It reconciles BTP Manager Secrets on Kyma runtimes. It can do it in two ways: 
+- by implementation with the usage of Runtime Watcher, which sends events about changes of the Secret from a Kyma runtime to KEB in real time.
```
Comment (grego952) [inline docs/03-17-runtime-reconciler.md:8]:
```suggestion
- by implementation with the usage of Runtime Watcher, which sends events about changes of the Secret from a Kyma runtime to Kyma Environment Broker (KEB) in real time. 
```

File: docs/08-01-provisioning-kyma-environment.md [hunk 23]
```diff
@@ -1,6 +1,6 @@
-# Provision Kyma Runtime using KEB
+# Provision SAP BTP, Kyma runtime using KEB
```
Comment (grego952) [inline docs/08-01-provisioning-kyma-environment.md:1]:
```suggestion
# Provision SAP BTP, Kyma runtime using Kyma Environment Broker
```

File: docs/08-01-provisioning-kyma-environment.md [hunk 24]
```diff
@@ -1,6 +1,6 @@
...
 
-This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environment Broker.
+This tutorial shows how to provision SAP BTP, Kyma runtime on Azure using Kyma Environment Broker.
 
 ## Prerequisites
```
Comment (grego952) [inline docs/08-01-provisioning-kyma-environment.md:5]: 1. The first link in the Prerequisites is broken, and the second one leads to the wrong page.
2. Please change all instances of Kyma Environment Broker to KEB (except the first one, of course)

File: docs/08-01-provisioning-kyma-environment.md [hunk 25]
```diff
@@ -1,6 +1,6 @@
-# Provision Kyma Runtime using KEB
+# Provision SAP BTP, Kyma runtime using KEB
 
-This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environment Broker.
+This tutorial shows how to provision SAP BTP, Kyma runtime on Azure using Kyma Environment Broker.
```
Comment (grego952) [inline docs/08-01-provisioning-kyma-environment.md:3]:
```suggestion
This tutorial shows how to provision SAP BTP, Kyma runtime on Azure using Kyma Environment Broker (KEB).
```

File: docs/08-02-deprovisioning-kyma-environment.md [hunk 26]
```diff
@@ -1,6 +1,6 @@
-# Deprovision Kyma Runtime using KEB
+# Deprovision SAP BTP, Kyma runtime using KEB
```
Comment (grego952) [inline docs/08-02-deprovisioning-kyma-environment.md:1]:
```suggestion
# Deprovision SAP BTP, Kyma runtime using Kyma Environment Broker
```

File: docs/08-02-deprovisioning-kyma-environment.md [hunk 27]
```diff
@@ -1,6 +1,6 @@
...
 
-This tutorial shows how to deprovision Kyma Runtime on Azure using Kyma Environment Broker.
+This tutorial shows how to deprovision SAP BTP, Kyma runtime on Azure using Kyma Environment Broker.
```
Comment (grego952) [inline docs/08-02-deprovisioning-kyma-environment.md:3]:
```suggestion
This tutorial shows how to deprovision SAP BTP, Kyma runtime on Azure using Kyma Environment Broker (KEB).
```

File: docs/08-02-deprovisioning-kyma-environment.md [hunk 28]
```diff
@@ -17,7 +17,7 @@ This tutorial shows how to deprovision Kyma Runtime on Azure using Kyma Environm
    export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
 
-3. Make a call to the Kyma Environment Broker to delete a Runtime on Azure.
+3. Make a call to the Kyma Environment Broker to delete a Kyma runtime on Azure.
```
Comment (grego952) [inline docs/08-02-deprovisioning-kyma-environment.md:20]:
```suggestion
3. Make a call to KEB to delete a Kyma runtime on Azure.
```

File: docs/08-05-orchestrate-kyma-upgrade.md [hunk 29]
```diff
@@ -1,6 +1,6 @@
 # Orchestrate Kyma upgrade
 
-This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
+This tutorial shows how to upgrade SAP BTP, Kyma runtime using Kyma Environment Broker.
```
Comment (grego952) [inline docs/08-05-orchestrate-kyma-upgrade.md:3]:
```suggestion
This tutorial shows how to upgrade SAP BTP, Kyma runtime using Kyma Environment Broker (KEB).
```

File: docs/08-05-orchestrate-kyma-upgrade.md [hunk 30]
```diff
@@ -1,6 +1,6 @@
...
 
 ## Prerequisites
```
Comment (grego952) [inline docs/08-05-orchestrate-kyma-upgrade.md:5]: Please change all instances of Kyma Environment Broker to KEB (except the first one, of course)

File: docs/08-05-orchestrate-kyma-upgrade.md [hunk 31]
```diff
@@ -16,14 +16,14 @@ This tutorial shows how to upgrade Kyma Runtime using Kyma Environment Broker.
    export AUTHORIZATION_HEADER="Authorization: Bearer $ID_TOKEN"
    ```
 
-2. Make a call to the Kyma Environment Broker to orchestrate the upgrade. You can select specific Runtimes to upgrade using the following selectors:
+2. Make a call to the Kyma Environment Broker to orchestrate the upgrade. You can select specific Kyma runtimes to upgrade using the following selectors:
```
Comment (grego952) [inline docs/08-05-orchestrate-kyma-upgrade.md:19]:
```suggestion
2. Make a call to KEB to orchestrate the upgrade. You can select specific Kyma runtimes to upgrade using the following selectors:
```

File: README.md [hunk 32]
```diff
@@ -1,38 +1,52 @@
...
+Kyma Environment Broker (KEB) is a component that allows you to provision [SAP BTP, Kyma runtime](https://kyma-project.io/#/?id=kyma-and-sap-btp-kyma-runtime) on clusters provided by third-party providers. In the process, KEB first uses Provisioner to create a cluster. Then, it uses Reconciler and Lifecycle Manager to install Kyma runtime on the cluster.
+
+## Configuration
+
+KEB binary allows you to override some configuration parameters. You can specify the following environment variables:
+
+| Name | Description | Default value |
+|-----|---------|:--------:|
+| **APP_PORT** | Specifies the port on which the HTTP server listens. | `8080` |
+| **APP_PROVISIONING_DEFAULT_GARDENER_SHOOT_PURPOSE** | Specifies the purpose of the created cluster. The possible values are: `development`, `evaluation`, `production`, `testing`. | `development` |
```
[the remark this answers is not in the document]
Comment (IwonaLanger) [reply README.md:15]: The period marks the end of the list in this column. `development` is in the next column.

File: docs/03-02-runtime-components.md (hunk 2 above)
Comment (IwonaLanger) [reply docs/03-02-runtime-components.md:18]: Thanks for spotting this one! :)

File: docs/03-02-runtime-components.md (hunk 3 above)
Comment (IwonaLanger) [reply docs/03-02-runtime-components.md:20]: Yes, that's right 🙇‍♀️

File: docs/03-10-orchestration.md (hunk 19 above)
Comment (IwonaLanger) [reply docs/03-10-orchestration.md:5]: Yes, you're right. It should be `#upgrade-kyma`.

File: docs/03-11-swagger.md [hunk 33]
```diff
@@ -6,17 +6,7 @@ The Swagger UI static files are copied from the [official source](https://github
...
 KEB uses a Swagger schema file that is mounted as a volume to the Pod. You can find that schema [here](https://github.com/kyma-project/control-plane/blob/main/resources/kcp/charts/kyma-environment-broker/files/swagger.yaml). You can use templates in the Swagger schema file to configure it.
 
-You can either use Virtual Service or port-forward the Pod to expose and use the Swagger UI.
-
-## Use Virtual Service
-
-Open the KEB APIs in the desired environment:
-- [DEV environment](https://kyma-env-broker.cp.dev.kyma.cloud.sap/) 
-- [STAGE environment](https://kyma-env-broker.cp.stage.kyma.cloud.sap/)
-
```
Comment (piotrmiskiewicz) [inline docs/03-11-swagger.md:16]: why it is deleted?

File: docs/03-15-trial-cleanup-cronjob.md [hunk 34]
```diff
@@ -1,7 +1,7 @@
-# Trial Cleanup Job
+# Trial Cleanup CronJob
 
-Trial Cleanup Job is a Job that makes the SKR instances with the `trial` plan expire 14 days after their creation.
-Expiration means that the SKR instance is suspended and the `expired` flag is set.
+Trial Cleanup CronJob is a Job that makes the SAP BTP, Kyma runtime instances with the `trial` plan expire 14 days after their creation.
```
Comment (piotrmiskiewicz) [inline docs/03-15-trial-cleanup-cronjob.md:3]: I know that the naming convention has some rules, but replacing all `SKR` with SAP BTP, Kyma runtime looks ugly. Do we have a possibility to change it?

Comment (piotrmiskiewicz) [reply docs/03-15-trial-cleanup-cronjob.md:3]: maybe we can replace `SKR` with `Kyma runtime`

File: README.md (hunk 32 above)
[the remark this answers is not in the document]
Comment (grego952) [reply README.md:15]: Ooops, sorry I missed that :D

**Review by grego952 — APPROVED** [review]

File: docs/03-15-trial-cleanup-cronjob.md (hunk 34 above)
Comment (IwonaLanger) [reply docs/03-15-trial-cleanup-cronjob.md:3]: I tried to use "SAP BTP, Kyma runtime" only once per doc (the first time) and then kept using "Kyma runtime".

File: docs/03-11-swagger.md (hunk 33 above)
Comment (piotrmiskiewicz) [reply docs/03-11-swagger.md:16]: ok, now I see, it is about links to our environments :)

**Review by piotrmiskiewicz — DISMISSED** [review]

**Review by grego952 — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**